node/src/effect/announcements.rs

```diff
@@ -122,6 +124,15 @@ pub enum ConsensusAnnouncement {
     Finalized(Box<FinalizedBlock>),
     /// A linear chain block has been handled.
     Handled(Box<BlockHeader>),
+    /// An equivocation has been detected.
+    Equivocation {
+        /// The era in which the equivocation is detected
+        era_id: EraId,
+        /// The public key of the equivocator
+        public_key: Box<PublicKey>,
```

Comment:
Does this need boxed?

Reply:
Yes, the event size reached 216 bytes, which spat out a warning. Should I box all three?

Reply:
No, the others are small.